addon/components/frost-table-cell.js

```diff
@@ -18,7 +18,9 @@ export default Component.extend({
   // == PropTypes =============================================================
 
   propTypes: {
+    onCallback: PropTypes.func.required,
     cellRenderer: PropTypes.any,
+    colIndex: PropTypes.number,
```

Comment:
It does, but I can't think of a good reason for the cell renderer to have it -- I'll remove both.